brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post layout - Issue with the custom taxonomy

### Description

- Added default value to Taxonomy.
- https://app.startinfinity.com/b/Mi5zu3VRVzk/5s2b2cSP7a5/a7d6876e-b6f1-41ca-86d2-ab0d8a99fe11?view=0f0c95d5-d336-4bdd-bb8b-f7cfb212b3d1&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Add CPT and add taxonomy and category to the CPT. Drag and drop Post Grid on a page.
- Change Post Type to CPT set taxonomy to category or Tags save and refresh the page.
- Then Change Post Type to Post and see if the category list is visible or not.
- Now switch to `post-layout-taxonomy-issue` and refresh the page and switch to Post.
- You will see the Category/Tag list.
- Screencast - https://share.getcloudapp.com/BludpdGJ

### Checklist:
